bot/client.js

```diff
@@ -57,4 +57,17 @@ client.on('message', (message) => {
   }
 });
 
+client.on('guildMemberAdd', (guildMember) => {
+  const channel = guildMember.guild.channels.find('name', 'general');
+  if (!channel) {
+    util.log('Channel Not Found', channel, 3);
+  } else {
+    // Announce new member in channel
+    channel.send(`Attention everyone! ${guildMember.user.username} has just joined the Armada!`);
+
+    // Send a welcome message to the user with terms of service
+    guildMember.send(`Welcome to the server ${guildMember.user.username}!\r\nYou can view our terms of service here!`);
```

Comment:
Narrowly scoped logic. This does satisfy the requirements of the assignment, but does so inefficiently. It would be better to evoke the controller built in welcomeController.